pkg/cloudmap/resource_manager.go

```diff
@@ -385,3 +390,32 @@ func (m *defaultResourceManager) isCloudMapServiceOwnedByVirtualNode(ctx context
 func (m *defaultResourceManager) buildCloudMapServiceSummaryCacheKey(nsSummary *servicediscovery.NamespaceSummary, serviceName string) string {
 	return fmt.Sprintf("%s/%s", awssdk.StringValue(nsSummary.Id), serviceName)
 }
+
+func (m *defaultResourceManager) getClusterNodeInfo(ctx context.Context) map[string]nodeAttributes {
+	var nodeInfoByName map[string]nodeAttributes
+	nodeList := &corev1.NodeList{}
+	if err := m.k8sClient.List(ctx, nodeList); err != nil {
+		return nodeInfoByName
+	}
+
+	m.log.V(1).Info("Nodes ", "count: ", len(nodeList.Items))
```

Comment:
better to be 
`m.log.V(1).Info("listed nodes", "count", len(nodeList.Items))`. don't use ":" :D